src/utils/channel_utils.js

```diff
@@ -647,3 +647,11 @@ export function filterChannelsMatchingTerm(channels: Array<Channel>, term: strin
             displayName.startsWith(lowercasedTerm);
     });
 }
+
+export function isManuallyUnread(channelId: ?string, state: GlobalState): boolean {
+    if (channelId) {
+        const isUnread = state.entities.channels.manuallyUnread[channelId];
+        return typeof (isUnread) !== 'undefined' && isUnread;
```

Comment:
I think you can replace this with
```suggestion
        return Boolean(isUnread);
```
to simplify it